cmd/hotkey-api/main.go

```
@@ -11,41 +12,114 @@ import (
 
 	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
 	defer stop()
 
-	api := app.NewAPI(cfg, log)
+	// Initialize Database
+	db, err := sql.Open("postgres", cfg.DatabaseURL)
+	if err != nil {
+		logSlog.Error("failed to open database", "error", err)
+		panic(err)
+	}
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)